inject.js

```diff
@@ -868,11 +876,25 @@ function handleDrag(video, controller, e) {
     style.top = initialControllerXY[1] + dy + "px";
   };
 
+  function handleReposition(video, controller) { // 481_NEW: handling reposition
+    const shadowController = controller.shadowRoot.querySelector('#controller');
+    // Find nearest parent of same size as video parent.
+    var parentElement = controller.parentElement;
+    while (parentElement.parentNode &&
+      parentElement.parentNode.offsetHeight === parentElement.offsetHeight &&
+      parentElement.parentNode.offsetWidth === parentElement.offsetWidth) {
+      parentElement = parentElement.parentNode;
+    }
+
+    let style = shadowController.style;
```

Comment:
Wait, now that I'm looking at this.. it's tricky, I don't think 0 is the right value and, in fact, we have custom positions on various sites: https://github.com/igrigorik/videospeed/blob/master/inject.css

Hmm?